pkg/scheduler/scheduler.go

```diff
@@ -158,7 +158,7 @@ func (s *Scheduler) schedule(ctx context.Context) {
 	// 1. Get the heads from the queues, including their desired clusterQueue.
 	// This operation blocks while the queues are empty.
 	headWorkloads := s.queues.Heads(ctx)
-	// No elements means the program is finishing.
+	// No elements, means the program is finishing.
```

Comment:
```suggestion
	// No elements means the program is finishing.
```

Reply:
How about this:
If there are no elements, it means that the program is finishing.

Reply:
SGTM